doc/nrf/releases/release-notes-latest.rst

```diff
@@ -88,6 +88,12 @@ BSD library
 
 * Added information about low accuracy mode to the :ref:`nrfxlib:gnss_extension` documentation.
 
+Trusted Firmware-M:
+-------------------
+
+Added a simple sample that demonstrates how to integrate TF-M in an application.
```

Comment:
```suggestion
* Added a simple sample that demonstrates how to integrate TF-M in an application.
```

Reply:
Sure about that?

Reply:
No objection, but wanted to check, because at the same level, `sdk-nrfxlib` does not have a body in bullet-point format

Reply:
Yes - sdk-nrfxlib has an introduction for the following subsections, while this TF-M section contains the list of changes. So while they are both the same level of headings, they have different content.

Reply:
OK changing this now, then.
Done